contrib/localnet/scripts/start-zetacored.sh

```diff
@@ -245,7 +245,7 @@ then
     .app_state.mint.params.mint_denom = "azeta" |
     .app_state.evm.params.evm_denom = "azeta" |
     .app_state.emissions.params.ballot_maturity_blocks = "30" |
-    .app_state.fungible.systemContract.gateway_gas_limit = "4000000" |
+    .app_state.fungible.systemContract.gateway_gas_limit = "1500000" |
```

Comment:
```suggestion
```
Let's just removing, I changed the logic in the other PR to set it through the chain params